components/tabzilla.jsx

```diff
@@ -0,0 +1,64 @@
+// This is a Tabzilla "lazy-loader" for sites that don't use jQuery.
+// Because Tabzilla requires jQuery and jQuery is relatively large,
+// we delay loading it until the last possible moment, when the user
+// has expressed intent to explore Tabzilla.
+//
+// Only one instance of this component is expected to ever be on the
+// page at once, and it's expected to exist for the lifetime of the
+// whole page.
+//
+// This component does require the following stylesheet to be loaded,
+// however:
+//
+//   //mozorg.cdn.mozilla.net/media/css/tabzilla-min.css
+
+var React = require('react');
+
+// This is currently the minimum version of jQuery that Tabzilla requires
+// to load.
+var JQUERY = '//mozorg.cdn.mozilla.net/media/js/libs/jquery-1.7.1.min.js';
```

Comment:
Hmm, do you mean if there's a jQuery conflict with Tabzilla, or with other code that uses jQuery on the Teach site? One nice thing about Tabzilla is that if it *does* decide to load jQuery itself, it automatically noconflict's *its* version of jQuery to make sure it doesn't interfere with the one already on the page, which is why I figured I didn't need to do anything from my end.